index.test.js

```diff
@@ -1216,14 +1217,12 @@ describe('Deploy to ECS', () => {
 
     test('run task', async () => {
         core.getInput = jest
-            .fn()
-            .mockReturnValueOnce('task-definition.json')  // task-definition
-            .mockReturnValueOnce('')                      // service
-            .mockReturnValueOnce('')                      // cluster
-            .mockReturnValueOnce('')                      // wait-for-service-stability
-            .mockReturnValueOnce('')                      // wait-for-minutes
-            .mockReturnValueOnce('')                      // force-new-deployment
-            .mockReturnValueOnce('true');                 // run-task
+            .fn(input => {
+                return {
+                    'task-definition': 'task-definition.json',
+                    'run-task': 'true',
+                }[input];
+            });
```

Comment:
In case you're wondering what's going on here - I've removed `jest` "magic" and replaced it with an actual object that is easier to test against. Same in several other areas.